docs/rules/ssr/no-unsupported-ssr-properties.md

```diff
@@ -0,0 +1,65 @@
+# Disallow access of properties on this during SSR (`lwc/no-unsupported-ssr-properties`)
```

Comment:
Since we moved the rules under 'ssr' folder, should we remove all references to 'ssr' in their names?

Reply:
Updated

Reply:
@abhagta-sfdc , could you check all references to the "ssr" in file names and make sure we remove them as they already should be under the "ssr" folder?

Reply:
The rules with ssr in their filenames have already been moved to ssr folder with these three rules as an exception :
no-unsupported-ssr-properties,no-node-env-in-ssr, no-restricted-browser-globals-during-ssr as these have been already released in previous version of the plugin.After discussing this with @lpomerleau ,we came to the conclusion that for now we will keep these rules but mark them as deprecated.With coming new versions we can remove these rules.